src/pages/developers/learning-tools.tsx

```diff
@@ -1,78 +1,103 @@
 // Library imports
 import React, { useEffect, useState } from "react"
-import styled from "@emotion/styled"
 import { graphql, PageProps } from "gatsby"
 import { useTranslation } from "gatsby-plugin-react-i18next"
 import { shuffle } from "lodash"
 // Component imports
+import { Box, Flex } from "@chakra-ui/react"
 import PageMetadata from "../../components/PageMetadata"
 import Translation from "../../components/Translation"
 import ButtonLink from "../../components/ButtonLink"
 import InfoBanner from "../../components/InfoBanner"
 import CalloutBanner from "../../components/CalloutBanner"
-import { Content, Page } from "../../components/SharedStyledComponents"
 import FeedbackCard from "../../components/FeedbackCard"
 import LearningToolsCardGrid from "../../components/LearningToolsCardGrid"
 // Util imports
 import { getImage } from "../../utils/image"
 // Type imports
-import { Context, LearningTool } from "../../types"
+import { ChildOnlyProp, Context, LearningTool } from "../../types"
 
-// Styled components
-const StyledPage = styled(Page)`
-  margin-top: 4rem;
-`
+const StyledPage = (props: ChildOnlyProp) => (
+  <Flex
+    flexDirection="column"
+    alignItems="center"
+    w="100%"
+    ml="auto"
+    mr="auto"
+    mb={16}
+    mt={16}
+    {...props}
+  ></Flex>
+)
 
-const Header = styled.header`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  text-align: center;
-  max-width: 896px;
-  padding: 0 2rem;
-`
-const H1 = styled.h1`
-  margin: 2rem 0 0;
-  margin-top: 0;
-  color: ${(props) => props.theme.colors.text};
-  font-style: normal;
-  font-family: ${(props) => props.theme.fonts.monospace};
-  text-transform: uppercase;
-  font-weight: 600;
-  font-size: 2rem;
-  line-height: 1.4;
-  text-align: center;
-`
+const Header = (props: ChildOnlyProp) => (
+  <Flex
+    flexDirection="column"
+    alignItems="center"
+    textAlign="center"
+    maxW="896px"
+    p={0}
+    pl={8}
+    pr={8}
+    {...props}
+  />
+)
 
-const Subtitle = styled.h2`
-  font-size: 1.25rem;
-  line-height: 1.4;
-  font-weight: 400;
-  color: ${(props) => props.theme.colors.text300};
-  max-width: 55ch;
-  margin-bottom: 0.5rem;
-  margin-top: 1rem;
-`
+const H1 = (props: ChildOnlyProp) => (
+  <Flex
```

Comment:
This should be an H1

```suggestion
  <Heading
    as="h1"
```